### PR TITLE
Remove set()

### PR DESCRIPTION
**Pull request by SHeimlich**
Step 4 of the set() to store() fun.

### CODE REVIEW

File: intellij-plugin/bin/main/META-INF/plugin.xml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<idea-plugin>
```
Comment (piotrswigon) [inline intellij-plugin/bin/main/META-INF/plugin.xml]: unrelated?

Comment (SHeimlich) [reply intellij-plugin/bin/main/META-INF/plugin.xml]: Gah! Fixed.
